packaging/rpm/make-rpm.sh

```diff
@@ -62,7 +62,7 @@ EOF
   # it yields w19.zstdio, which means compression level 19 using zstd algorithm. To
   # speed this up it runs w19T8.zstdio, which is the 8 thread version of the same
   # algorithm.
```

Comment:
It looks like this comment will also need to be updated.

I wonder if we need any compression at all if this is just for CI?